index.html

```diff
@@ -50,6 +51,7 @@
       .data { font-family: 'Major Mono Display', monospace; }
 
       .fill-base { fill: #eb9; }
+      .fill-brass { fill: #dc9018; }
```

Comment:
needed to add this as a fill option, but didn't end up using it for these changes